packages/provider/src/language-model/v1/language-model-v1-logprobs.ts

```diff
@@ -0,0 +1,8 @@
+export type LanguageModelV1LogProbs = {
+  token: string;
+  logprob: number;
+  top_logprobs: {
```

Comment:
We use camelCase for anything that is developer facing. Please change to `topLogprobs` and map the data from OpenAI.